src/main/resources/PortadaAplicacion.fxml

```diff
@@ -0,0 +1,248 @@
+                  <VBox alignment="CENTER" layoutX="10.0" layoutY="10.0" onMouseClicked="#HandleDone" prefHeight="200.0" prefWidth="100.0" styleClass="bottom-aplication-1" stylesheets="@usuario.css" GridPane.columnIndex="1">
+                     <GridPane.margin>
+                        <Insets left="-1.0" right="-1.0" />
+                     </GridPane.margin>
+                     <children>
+                        <ImageView fitHeight="70.0" fitWidth="70.0" pickOnBounds="true" preserveRatio="true">
+                           <image>
+                              <Image url="@temperatura-botones-inferiores.png" />
+                           </image>
+                        </ImageView>
+                     </children>
+                  </VBox>
+                  <VBox alignment="CENTER" layoutX="194.0" layoutY="10.0" onMouseClicked="#HandleDone" prefHeight="200.0" prefWidth="100.0" styleClass="bottom-aplication-center" stylesheets="@usuario.css" GridPane.columnIndex="2">
+                     <GridPane.margin>
+                        <Insets />
+                     </GridPane.margin>
+                     <children>
+                        <ImageView fitHeight="70.0" fitWidth="70.0" pickOnBounds="true" preserveRatio="true">
+                           <image>
+                              <Image url="@logo-balanpy.png" />
+                           </image>
+                        </ImageView>
+                     </children>
+                  </VBox>
+                  <VBox alignment="CENTER" layoutX="250.0" layoutY="10.0" onMouseClicked="#HandleDone" prefHeight="200.0" prefWidth="100.0" styleClass="bottom-aplication-1" stylesheets="@usuario.css" GridPane.columnIndex="3">
+                     <GridPane.margin>
+                        <Insets left="-1.0" right="-1.0" />
+                     </GridPane.margin>
+                     <children>
+                        <ImageView fitHeight="70.0" fitWidth="70.0" pickOnBounds="true" preserveRatio="true">
+                           <image>
+                              <Image url="@veterinario-botones-inferiores.png" />
+                           </image>
+                        </ImageView>
+                     </children>
+                  </VBox>
+                  <VBox alignment="CENTER" layoutX="370.0" layoutY="10.0" onMouseClicked="#HandleDone" prefHeight="200.0" prefWidth="100.0" styleClass="bottom-aplication-1" stylesheets="@usuario.css" GridPane.columnIndex="4">
+                     <children>
+                        <ImageView fitHeight="70.0" fitWidth="70.0" pickOnBounds="true" preserveRatio="true">
+                           <image>
+                              <Image url="@usuario-botones-inferiores.png" />
+                           </image>
+                        </ImageView>
+                     </children>
+                  </VBox>
+               </children>
+            </GridPane>
+         </children>
+      </VBox>
```

Comment:
Este componente lo vamos a utilizar en varias pantallas de la aplicación, en lugar de copiar y pegar estas líneas en múltiples lugares sería bueno extraerlo en su propio fxml (algo como `BarraTareas.fxml`) he incluirlo en las pantallas necesarias usando un tag `<fx:include>`

Aquí hay una pequeña muestra de cómo se podría hacer por si te animas, caso contrario lo hago en un PR aparte: http://blog.buildpath.de/fxml-composition-how-to-get-the-controller-of-an-included-fxml-view-nested-controllers/